packages/@aws-cdk/aws-iotevents-actions/lib/iotevents-set-variable-action.ts

```diff
@@ -0,0 +1,25 @@
+import * as iotevents from '@aws-cdk/aws-iotevents';
+import { Construct } from 'constructs';
+
+/**
+ * The action to create a variable with a specified value.
+ */
+export class IoteventsSetVariableAction implements iotevents.IAction {
```

Comment:
What's with the weird name? Why the weird `Iotevents` prefix?

I think this should be called just `SetVariableAction`.

I _could_ be convinced to `IotSetVariableAction` or `IotEventsSetVariableAction`, but in my opinion, the `Iot` / `IotEvents` prefix doesn't add anything here.